cogs/get_token_authorisation.py

```diff
@@ -85,14 +86,13 @@ async def get_token_authorisation(self, ctx: TeXBotApplicationContext) -> None:
 
         user_name: bs4.Tag | bs4.NavigableString | int | None = profile_section_html.find("h1")
 
-        NO_PROFILE_DEBUG_MESSAGE: Final[str] = "Found user profile but couldn't find their name!"  # noqa: E501
-
         if not isinstance(user_name, bs4.Tag):
+            NO_PROFILE_DEBUG_MESSAGE: Final[str] = "Found user profile but couldn't find their name!"  # noqa: E501
```

Comment:
String should be split onto multiple lines rather than ignoring the line-length error:
```python
NO_PROFILE_DEBUG_MESSAGE: Final[str] = (
    "FOUND user profile but couldn't find their name!"
)
```